src/abstractdataframe/selection.jl

```diff
@@ -254,17 +257,28 @@ function normalize_selection(idx::AbstractIndex,
                 end
             end
     end
+
     if lls isa AbstractString
-        r = Symbol(lls)
+        combine_target_col = Symbol(lls)
     elseif lls isa AbstractVector{<:AbstractString}
-        r = Symbol.(lls)
+        combine_target_col = Symbol.(lls)
```

Comment:
when you add tests can you please add a test that will cover this line? It is unrelated to the PR, but since you change the line we should cover it with tests.

Reply:
Sure

Reply:
Added